src/types/recruiter/recruiter.ts

```diff
@@ -0,0 +1,7 @@
+export interface RecruiterInterface {
+    id: object;
```

Comment:
Here in id is object because the relation required me to do so.

Reply:
Intresting...
Let's leave it. We will test it more after merge to develop.